addons/debugger/timing/TimingManager.js

```diff
@@ -0,0 +1,53 @@
+class TimingManager {
+  constructor(settings, config, profiler) {
+    this.dummyProfiler = profiler;
+    this.settings = settings;
+    this.config = config;
+
+    this.timers = {};
```

Comment:
Nitpick, but this should probably be a `Map`. Because it's an object, creating a timer named `__proto__` will crash the VM.

Reply:
this.timers = Object.create(null); will fix with this very simple fix.